specs/deneb/p2p-interface.md

```diff
@@ -188,16 +217,23 @@ Request Content:
 ```
 
 Response Content:
+
+```python
+class BlobSidecars(Container):
+  block_root: Root
```

Comment:
`block_root` is in each sidecar.

Do we need this in the container? or can we just force each blob_sidecar to have the same `block_root`. In fact, do we need the container at all? I guess it makes it easier to specify.

Reply:
Yeah, since the request counts blocks (and not blobs), it's easier to keep the responses grouped by block also, making it easier to write the spec. 

Agree it's somewhat arbitrary so if there's any preference other than this logical grouping, I'm fine with whatever.

Reply:
Sleeping on this helped me see the brilliance of flattening the list and returning plain `BlobSidecar` instances: we don't have to come up with a name for the container! da34af97d4b6575131e67c2bf22acb6de4f0951d